pkg/loop/internal/pb/ccipocr3/models.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/ccipocr3;ccipocr3pb";
+
+package loop.internal.pb.ccipocr3;
+
+// CCIPMsg is a gRPC adapter to [github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3.CCIPMsg]
+message CCIPMsg {
```

Comment:
With the family-agnostic ramp work (currently in spike and ongoing) this may be affected. It may end up to be something like this:

```proto
message EVM2AnyMessage {
  // ... fields
}

message Any2EVMMessage {
  // .. fields
}

message CCIPMessage {
  oneof message_payload {
    EVM2AnyMessage evm_2_any_message = 1;
    Any2EVMMessage any_2_evm_message = 5;
  }
}
```

Reply:
yeap, exactly